### PR TITLE
feat: Show rmrk version in item description

### LINKED ISSUE #5681: Show rmrk version in item description
https://deploy-preview-5654--koda-canary.netlify.appl/ksm/gallery/16848738-7092505797a30ccf48-COLORED_BEASTS-COLORED_BEAST_COMMON_6-00000003

image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5681
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
`/ksm/gallery/13328952-54bbd380dc3baaa27b-EVRSOULS-Ranger-00000922`
image

`/rmrk/gallery/17483893-9AFFD71D65E3DBBF29-KODA_VILLA-KODA_VILLAGE_2-0000000000000002`
image

### CODE REVIEW

**Review by vikiival — APPROVED** [review]

File: components/gallery/GalleryItemDescription.vue [hunk 1]
```diff
@@ -105,6 +111,7 @@ const { urlPrefix } = usePrefix()
 const { nft, nftMimeType, nftMetadata, nftImage, nftAnimation } =
   useGalleryItem()
 const activeTab = ref('0')
+const { isV2 } = useRmrkVersion()
```
Comment (vikiival) [inline components/gallery/GalleryItemDescription.vue:114]:
```suggestion
const { version } = useRmrkVersion()
```

File: components/gallery/GalleryItemDescription.vue [hunk 2]
```diff
@@ -55,6 +55,12 @@
         <p>{{ $t('tabs.tabDetails.blockchain') }}</p>
         <p>{{ urlPrefix }}</p>
       </div>
+      <div
+        v-if="urlPrefix === 'rmrk' || urlPrefix === 'ksm'"
+        class="is-flex is-justify-content-space-between">
+        <p>{{ $t('tabs.tabDetails.version') }}</p>
+        <p>{{ isV2 ? '2.0' : '1.0' }}</p>
+      </div>
```
Comment (vikiival) [inline components/gallery/GalleryItemDescription.vue:63]:
```suggestion
      <div
        v-if="version"
        class="is-flex is-justify-content-space-between">
        <p>{{ $t('tabs.tabDetails.version') }}</p>
        <p>{{ version }}</p>
      </div>
```

**Review by vikiival — CHANGES_REQUESTED** [review]

**Comment by exezbcz** [comment]
image
it shows 1.0.0 on my end, is that alright? (The 3 digit version format) 


for rmrk2 as well
image

**Comment by vikiival** [comment]
Yup It's correct

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 36.61 USD/KSM ~ 0.273 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xb6e8c99ead509daf936ca3932f1c9798ad2f7579f951c1820dea24b6f0dbcc15](https://kusama.subscan.io/extrinsic/0xb6e8c99ead509daf936ca3932f1c9798ad2f7579f951c1820dea24b6f0dbcc15)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**